libc/test/src/fcntl/fcntl_test.cpp

```diff
@@ -162,4 +163,23 @@ TEST(LlvmLibcFcntlTest, UseAfterClose) {
   ASSERT_THAT(LIBC_NAMESPACE::close(fd), Succeeds(0));
   ASSERT_EQ(-1, LIBC_NAMESPACE::fcntl(fd, F_GETFL));
   ASSERT_ERRNO_EQ(EBADF);
+  LIBC_NAMESPACE::libc_errno = 0;
```

Comment:
```suggestion
  // Clear errno for subsequent unit tests.
  LIBC_NAMESPACE::libc_errno = 0;
```

Reply:
I just noticed that other tests tend to zero libc_errno before running any operations.  We should clear errno first, for tests that need it, rather than reset it after tests that touch it.

Reply:
Okay, fixed. And I'm curious, is this just a matter of convention?

Reply:
Convention.  I think `UseAfterClose` and the other unit tests that use `ASSERT_ERRNO_` macros should be doing this, too.